API_Database.py

```diff
@@ -1,6 +1,7 @@
 import sqlite3 
 import API_Manager
```

Comment:
Is this import needed? Should the API_Database be independent of the API requests?

Reply:
No that is part of the code cleanup. I am hoping our team can work on that together for our meeting today.